server/tests/backends/sql_translator_integration_tests/test_sql_bigquery_translator_steps.py

```diff
@@ -0,0 +1,55 @@
+import json
+from os import environ
+
+import pandas as pd
+import pytest
+from google.cloud.bigquery import Client
+from google.oauth2 import service_account
+
+from tests.utils import assert_dataframes_equals, get_spec_from_json_fixture, retrieve_case
+from weaverbird.backends.pypika_translator.dialects import SQLDialect
+from weaverbird.backends.pypika_translator.translate import translate_pipeline
+from weaverbird.pipeline import Pipeline
+
+print(f"\n\n\nCREDS {environ['GOOGLE_BIG_QUERY_CREDENTIALS']}\n\n\n")
+
+credentials = service_account.Credentials.from_service_account_info(
+    info=json.loads(environ['GOOGLE_BIG_QUERY_CREDENTIALS'])
+)
```

Comment:
```suggestion
try:
    credentials = service_account.Credentials.from_service_account_info(
        info=json.loads(environ['GOOGLE_BIG_QUERY_CREDENTIALS'])
    )
except KeyError:
    pytestmark = pytest.mark.skip(reason='Missing environ variable "GOOGLE_BIG_QUERY_CREDENTIALS"')
```

Reply:
I'm against this. We want a misconfigured CI to fail, not to skip the tests that can't be run